values.yaml

```diff
@@ -36,6 +36,11 @@ coderd:
   # coderd.serviceAnnotations -- Extra annotations to apply to the coderd service.
   serviceAnnotations: {}
 
+  # coderd.httpsToHttp -- eliminates the external http port and routes traffic from
+  # the external https port to the internal http port. Useful for when the load balancer
+  # performs TLS termination (like Amazon's ACM)
+  httpsToHttp: false
```

Comment:
I've done a little bit of reading ([How do I terminate HTTPS traffic on Amazon EKS workloads with ACM?
](https://aws.amazon.com/premiumsupport/knowledge-center/terminate-https-traffic-eks-acm/)) but am by no means an expert on AWS.

It seems unusual to me that we'd be using a `LoadBalancer` spec when a frontend device is performing termination for us, and instead I'd have expected that we'd use an `Ingress` definition to do this? A Kubernetes LoadBalancer would be more like an Layer 3 (Network Load Balancer) and a Service with Ingress would be more like a Layer 7 device (Application Load Balancer)

Alternatively, is there a way to configure things manually so that we expose coderd as a ServiceIP (on the internal VPC network) and then manage the NLB/ALB outside of our Kube specs? Adding stuff to our Helm chart, especially in the absence of tests, increases complexity, so my preference is always to keep one-off customizations like this out of there, unless this is a very general thing with all of the clouds...

Reply:
It seems like an alternative to this implementation would be to make the full service spec configurable as users want, instead of using conditionals based on this `httpsToHttp` setting